tests/functional/test_tracing.py

```diff
@@ -212,3 +214,15 @@ def handler(event, context):
     result = handler({}, {})
     assert "testresult" in result
     assert "testresult2" in result
+
+
+def test_aiohttp_trace_config():
+    aiohttp_mock = Mock()
+    sys.modules["aiohttp"] = aiohttp_mock
+    from aws_lambda_powertools.tracing import aiohttp_trace_config
+
+    aiohttp_trace_config()
+
+    from aws_xray_sdk.ext.aiohttp.client import aws_xray_trace_config
+
+    assert aws_xray_trace_config.__doc__ == "aiohttp extension for X-Ray (aws_xray_trace_config)"
```

Comment:
```suggestion
    assert aiohttp_trace_config.__doc__ == "aiohttp extension for X-Ray (aws_xray_trace_config)"
```

I believe you got this backward -- You are asserting what we export has the right docstring, not what X-Ray exports per se.